src/sonic_ax_impl/mibs/__init__.py

```diff
@@ -43,9 +49,35 @@
 
 # IfIndex to OID multiplier for transceiver
 IFINDEX_SUB_ID_MULTIPLIER = 1000
+NET_NS_PATH = "/var/run/netns/"
 
 redis_kwargs = {'unix_socket_path': '/var/run/redis/redis.sock'}
 
+def setns(fd, nstype):
```

Comment:
>setns [](start = 4, length = 5)

These new features has nothing to do with `mibs`, and it may be reused outside this repo? Could you move into a common repo? Better to encapsulate a class.

Reply:
Removed this.